### PR TITLE
Replace production couch node 1

### PR DESCRIPTION
**Pull request by dannyroberts**
https://dimagi-dev.atlassian.net/browse/SAAS-14844

- Step 0: https://github.com/dimagi/commcare-cloud/pull/6108
- **Step 1: https://github.com/dimagi/commcare-cloud/pull/6116**
- Step 2: https://github.com/dimagi/commcare-cloud/pull/6117
- Step 3: https://github.com/dimagi/commcare-cloud/pull/6124
- Step 4: https://github.com/dimagi/commcare-cloud/pull/6110

This PR will be applied using Phase 2 of https://confluence.dimagi.com/display/saas/Replace+a+Couch+node+in+Couch+cluster (applied to just this one node replacement).

##### Environments Affected
production

### CODE REVIEW

File: environments/production/migrations/0009-move-to-3-jammy-nodes/couchdb-step-1.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+target_allocation:
```
Comment (AmitPhulera) [inline environments/production/migrations/0009-move-to-3-jammy-nodes/couchdb-step-1.yml:1]: Curious what this file is used for?

**Review by AmitPhulera — APPROVED** [review]

File: environments/production/migrations/0009-move-to-3-jammy-nodes/couchdb-step-1.yml (hunk 1 above)
Comment (dannyroberts) [reply environments/production/migrations/0009-move-to-3-jammy-nodes/couchdb-step-1.yml:1]: In https://confluence.dimagi.com/display/saas/Replace+a+Couch+node+in+Couch+cluster, it's the PLAN_FILE that is used in phase 2.

Comment (gherceg) [reply environments/production/migrations/0009-move-to-3-jammy-nodes/couchdb-step-1.yml:1]: Taking a stab at answering this cause why not 😅 I believe we reference this migration file when running `cchq ${ENV} migrate-couchdb ${PLAN_FILE} plan` in phase 2 of adding this node. And then it looks like that file is used to [allocate shards properly](https://github.com/dimagi/commcare-cloud/blob/2bb8d4bdd71ad8efc3c150c8fc03eeb5591ec47d/src/commcare_cloud/commands/migrations/couchdb.py#L321-L323) via couchdb-cluster-admin.

So then the syntax here is saying that we have three nodes and should make three copies of data across these nodes, which might be the same as saying 1 copy per node?

**Review by gherceg — APPROVED** [review]

File: environments/production/migrations/0009-move-to-3-jammy-nodes/couchdb-step-1.yml (hunk 1 above)
Comment (dannyroberts) [reply environments/production/migrations/0009-move-to-3-jammy-nodes/couchdb-step-1.yml:1]: Yeah but you said it better, Graham :)

Comment (AmitPhulera) [reply environments/production/migrations/0009-move-to-3-jammy-nodes/couchdb-step-1.yml:1]: Thanks @dannyroberts and @gherceg. That was super helpful :)

**Pull request merged**